jokers/burnt.lua

```diff
@@ -0,0 +1,52 @@
+SMODS.Enhancement({
+    key = "burnt",
+    atlas = "folly_enhancers",
+    pos = { x = 0, y = 0 },
+    min = 0.5,
+    max = 1.5,
+    replace_base_card = true,
+    no_rank = true,
+    no_suit = true,
+    overrides_base_rank = true,
```

Comment:
```suggestion
```
this is not really needed since `no_rank` is set to `true`.
Keep it if you prefer it being there.